xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_support.xsd

```diff
@@ -407,6 +407,16 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="vehicleLoadingPlace"/>
 			<xsd:enumeration value="handicappedLevelBoardingSector"/>
 			<xsd:enumeration value="multimodal"/>
+			<xsd:enumeration value="busOnStreet">
```

Comment:
That's what I was expecting from BusStop ... needs clarification

Reply:
added clarification.